client/components/mma/shared/asyncComponents/DefaultApiResponseHandler.tsx

```diff
@@ -0,0 +1,44 @@
+import type ResponseProcessor from './ResponseProcessor';
+
+export const JsonResponseHandler: ResponseProcessor = (response: Response) => {
+	return handleAResponses(response, (r: Response) => r.json());
+};
+
+export const TextResponseHandler: ResponseProcessor = (response: Response) => {
+	return handleAResponses(response, (r: Response) => r.text());
+};
+
+function handleAResponses(
```

Comment:
```suggestion
function handleAResponse(
```

Reply:
Definite typo. But need to take another look at this and decide whether response can be an array of responses